plugins/modules/jdiff.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2020, Network to Code (@networktocode) <info@networktocode.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""Ansible plugin definition for jdiff action plugin."""
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+---
+module: jdiff
+short_description: Ansible module for jdiff.
+version_added: '2.0.0'
+description:
+    - Ansible module wrapper on jdiff python library.
+requirements:
+    - jdiff
+author: Patryk Szulczewski (@pszulczewski)
+options:
+    check_type:
+        description:
+            - Check type supported by jdiff
```

Comment:
could we use `choices` to make life eaiser?

Reply:
I want this module to be just a wrapper, adding more check types in jdiff should not require any modification in this module. It should be transparent to jdiff.